app/views/users/_edit_form.html.erb

```diff
@@ -2,8 +2,15 @@
 <%= form_for @user, :url => {:controller => 'users', :action => 'update'}, :html => {:class => "row",:id => 'edit-form' } do |f| %>
   <%= hidden_field_tag "user[ui_update]", "true" %>
   <div class="col-md-10">
-    <% if f.error_messages != "" %>
-      <div class="alert alert-danger"><%= f.error_messages %></div>
+    <% if @model.errors.any? %>
```

Comment:
Aha! Yes, so here, the `@model` is a placeholder - we can replace it with `@user` and this should start to work! For all lines below as well. Sorry that was a bit ambiguous! This way it'll get the errors for the `@user` object. 

Let's try that! Great work so far!